### ISSUE TITLE
Tooltip without icon missing correct focus outline

### ISSUE THREAD

**Issue by emyarod** [IBM/carbon-components-react#2048]
When focusing on a tooltip without icon, Carbon focus outline is not applied and instead the default outline is shown

possible ticket for vanilla repo as well:

![image](https://user-images.githubusercontent.com/8265238/54613045-0b492100-4a28-11e9-9dd7-eb4fc479192a.png)

![image](https://user-images.githubusercontent.com/8265238/54613197-4c413580-4a28-11e9-92fa-659c719b37c5.png)
